frontend/ui/widget/inputdialog.lua

```diff
@@ -437,9 +458,17 @@ function InputDialog:init()
                 self._input_widget,
             },
             vspan_after_input_text,
-            buttons_container,
+            buttons_container
         }
     }
+
+    -- insert check button before the regular buttons
+    if self.check_button then
+        local nr_elements = #self.dialog_frame[1]
```

Comment:
```
$ rgrep nr_ frontend | wc -l
8
$ rgrep nb_ frontend | wc -l
136
$ rgrep num_ frontend | wc -l
40
```

Reply:
Maybe a false German friend "number=nummer" "nb=nr". I will change that.